### PR TITLE
Prepare for the 0.18 release

### PR DESCRIPTION
**Pull request by mateuszrzeszutek**
This needs to wait for #354 at least

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -226,41 +238,41 @@ You can now retrieve the session id in your JS code using `SplunkRumNative.getNa
...
   This can be used to provide customizations of the spans that are emitted by the library. Examples
   include: removing spans altogether from export, removing span attributes, changing span attributes
   or changing the span name. See the javadoc on the `SpanFilterBuilder` class for more details.
-- `slowRenderingDetectionPollInterval(Duration)` :
-  Set/change the default polling interval for slow/frozen render detection.
-  Default is 1000ms. Value must be positive.
-- `slowRenderingDetectionEnabled(boolean)` :
-  Disables the detection of slow frame renders. Enabled by default.
-- `diskBufferingEnabled(boolean)` :
+- `enableDiskBufferingEnabled()` :
```
Comment (breedx-splk) [inline README.md:275]:
```suggestion
- `enableDiskBuffering()` :
```

File: sample-app/build.gradle.kts [hunk 2]
```diff
@@ -49,7 +49,9 @@ android {
     }
 }
 
-val otelVersion = "1.16.0-alpha"
+val otelVersion = "1.18.0"
```
Comment (breedx-splk) [inline sample-app/build.gradle.kts:52]: 🙈

**Review by breedx-splk — APPROVED** [review]
Looks good to me. One small correction in the docs, otherwise great stuff!

**Review by theletterf — APPROVED** [review]

**Comment by mateuszrzeszutek** [comment]
> @mateuszrzeszutek Do we need to document the Volley integration in the official docs? I'd need a task for that if yes.

I believe we do.

**Comment by breedx-splk** [comment]
Superseded by #362

**Pull request closed without merging**